docs/0.1/building_grid.md

```diff
@@ -0,0 +1,34 @@
+# Building Grid
+
+<!--
+  Copyright (c) 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Local Builds
+
+To build Grid locally, run `cargo build` from the root directory. This command
+builds all of the Grid components, including `gridd` (the grid daemon),
+the CLI, and all of the smart contracts in the `contracts` directory.
+
+To build individual components, run `cargo build` in the component directories.
```

Comment:
Should we mention where the executable are placed once they're built (`./target/debug`)